frontend/test/metabase/scenarios/question/settings.cy.spec.js

```diff
@@ -165,13 +91,3 @@ function loadOrdersTable() {
   cy.contains("Sample Dataset").click();
   cy.contains("Orders").click();
 }
-
-// This isn't needed if there's only one db. In that case, clicking "Sample
-// Dataset" will actually take you back to select a db again.
-function maybeClickSampleDataset() {
```

Comment:
Looks like the removal of this is causing the linter and cypress tests to fail since it's imported and referenced elsewhere. Do we need to restore this or stop using it?